src/content/en/fundamentals/architecture/_toc.yaml

```diff
@@ -3,4 +3,17 @@ toc:
   section:
   - title: The App Shell Model
     path: /web/fundamentals/architecture/app-shell
-    
+  - title: "HowTo: Components"
+    section:
+    - title: "Intro"
+      path: /web/fundamentals/architecture/howto-components/howto-components
+    - title: "&lt;howto-radio-group&gt;"
```

Comment:
I just fixed a behaviour difference in DevSite vs staging (#4489). Merge that fix into this branch, then change these to be the unescaped versions. IE `- title: "<howto-radio-group>"`